eng/Build.props

```diff
@@ -117,6 +120,17 @@
         <ProjectToBuild Condition=" '$(BuildJava)' == 'true'" Include="@(JavaProjects)" Exclude="@(ProjectToExclude)" />
         <ProjectToExclude Condition=" '$(BuildJava)' != 'true'" Include="@(JavaProjects)" />
 
+        <!-- These projects have a transitive dependency on Microsoft.AspNetCore.Components.Web.JS exclude these projects if we are not building NodeJS -->
+        <!-- Do not skip these projects on the CI since we don't want to accidentally miss building artifacts -->
```

Comment:
Mention there's no reason to skip on the CI because the NodeJS artifacts are input

Reply:
Well... it's more for safeguarding. Even if we didn't build NodeJS artifacts as input I would keep this. Just so that any issues will fail loudly rather than silently not building some projects.